cmd/e2e-test/cleanup/cleanup.go

```diff
@@ -34,6 +34,13 @@ func NewCommand() *Command {
 	return &cmd
 }
 
+func NewCommandWithParams(resourcesFilePath string) *Command {
```

Comment:
Could you explain why we need this?
In your PR description you indicate that the current code "makes it harder to be called from other packages". But, aren't our canary tests using this same code already? We have service objects following the command pattern, the same ones being called from here and from the canaries.

So why do we need to make the actual flaggy command exportable? Does the new code that is supposed to import this have new requirements we didn't cover before?